index.d.ts

```diff
@@ -0,0 +1,6 @@
+import {Component} from 'react';
+interface IDocumentTitleProps {
+  title: string;
+}
+declare class DocumentTitle extends Component<IDocumentTitleProps, void> {}
```

Comment:
`void` is not valid in newer React types. You should use `{}` for back and forward compatibility.